doc/modules/mixture.rst

```diff
@@ -211,49 +211,68 @@ Cons
 
 .. _dpgmm:
 
-DPGMM: Infinite Gaussian mixtures
-=================================
+Dirichlet Process: Infinite Gaussian mixtures
+==============================================================
+
+The :class:`DirichletGaussianMixture` object implements a variant of the
+Gaussian mixture model with a variable (but bounded) number of components using
+the Dirichlet Process.
 
-The :class:`DPGMM` object implements a variant of the Gaussian mixture
-model with a variable (but bounded) number of components using the
-Dirichlet Process.
 This class doesn't require the user to choose the number of
 components, and at the expense of extra computational time the user
 only needs to specify a loose upper bound on this number and a
 concentration parameter.
 
 .. |plot_gmm| image:: ../auto_examples/mixture/images/sphx_glr_plot_gmm_001.png
    :target: ../auto_examples/mixture/plot_gmm.html
-   :scale: 48%
+   :scale: 31%
```

Comment:
I guess the [rendered page](https://27-50829930-gh.circle-artifacts.com/0//home/ubuntu/scikit-learn/doc/_build/html/stable/modules/mixture.html#dirichlet-process-infinite-gaussian-mixtures) is not as intended.